adafruit_cpython.py

```diff
@@ -1,26 +0,0 @@
-# SPDX-FileCopyrightText: 2017 Scott Shawcroft for Adafruit Industries
-#
-# SPDX-License-Identifier: MIT
-
-"""
-`adafruit_cpython`
-====================================================
```

Comment:
This is the title that was used before. I think you'll want something like this in colorsys.py.